beacon_node/network/src/status.rs

```
@@ -29,8 +29,21 @@ pub(crate) fn status_message<T: BeaconChainTypes>(beacon_chain: &BeaconChain<T>)
         finalized_checkpoint.root = Hash256::zero();
     }
 
-    let earliest_available_slot = beacon_chain.store.get_anchor_info().oldest_block_slot;
+    let earliest_available_data_column_slot = beacon_chain
+        .store
+        .get_data_column_custody_info()
+        .ok()
```

This is a biiit sketchy in that it swallows the DB read error, and then the logic below defaults to assuming we have full info.

Might be worth just adding a `NOTE` that we are making the assumption that `get_data_column_custody_info` won't fail

---

i've added a note